### DISCUSSION TITLE
Did not receive verificatio email when registering

### DISCUSSION THREAD

**Discussion by Kevin-free** [danny-avila/LibreChat#3008] [Troubleshooting]
### What happened?

The latest main branch, on your website https://librechat-librechat.hf.space/ and my local development, have not received the verification email.

p.s. I can’t receive it with other Gmail accounts either.     

### Steps to Reproduce

1. Click [sign up]  to register and fill in the information, such as: example@gmail.com
2. Click [continue] to be prompted to check the verification email and jump to the login page.
3. I checked the example@gmail.com email address and did not receive the verification email, nor did I click Resend.

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
My local log:


2024-06-08 10:55:01 info: [sendVerificationEmail] Verification link issued. [Email: example@gmail.com]
2024-06-08 10:56:16 error: [sendEmail] connect ETIMEDOUT 74.xxx.xxx.109:465
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
I'm not sure why email password reset / verification isn't working on our demo, I've set `ALLOW_UNVERIFIED_EMAIL_LOGIN=true` to bypass the requirement for verification in the meantime

(It's working from my home server deployment)

**Comment by danny-avila** [comment]
there’s an error, so the email failed. For security reasons, the UI will stay say that an email was sent. I tested this extensively so I think it may have to do with configuration.

Here are the settings I tested with:

```.env
EMAIL_SERVICE=
EMAIL_HOST=REDACTED
EMAIL_PORT=REDACTED
EMAIL_ENCRYPTION=tls
EMAIL_ENCRYPTION_HOSTNAME=
EMAIL_ALLOW_SELFSIGNED=
EMAIL_USERNAME=noreply@librechat.ai
EMAIL_PASSWORD=REDACTED
EMAIL_FROM_NAME=LibreChat
EMAIL_FROM=noreply@librechat.ai
```

I will try with gmail and see if it’s reproducible with that service. Since @fuegovic had issues I suspect there may be an IP restriction on smtp in certain environments?

Please also test the password reset to confirm if emails can get sent out at all in your environment.


https://www.librechat.ai/docs/configuration/authentication/password_reset

**Reply by fuegovic** [reply]
After further testing, it became clear that the issue with our hf demo stems from the "email service" we were using, which no longer offers a free tier, resulting in emails not being sent.

**Comment by danny-avila** [comment]
Also check this GitHub thread: https://github.com/nodemailer/smtp-server/issues/121

**Comment by danny-avila** [comment]
Ok I did a thorough review and tested with "gmail" with no issues. It's possible that the incoming/outgoing ports needed for SMTP may be blocked in your environments. I can definitely see that being the case for HuggingFace. I will test in a remote server soon.

If email fails, signup overall will now fail after this PR:
https://github.com/danny-avila/LibreChat/pull/3010

**Reply by fuegovic** [reply]
@Kevin-free, it's important to note that HuggingFace blocks ports 0-1024. Therefore, for email functionality, you must use a service that allows the use of the alternative SMTP port 2525.
